src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/ComponentExampleBox.razor

```diff
@@ -2,7 +2,7 @@
     <div class="box-header">
         <div class="header-title">@Title</div>
         <div class="header-btn-group">
-            <BitToggleButton Class="com-example-box-header-btn"
+            <BitToggleButton Class="com-example-box-header-btn com-example-box-header-btn-show-code"
```

Comment:
can't we use a better approach to apply the new styles?
for example, using the element hierarchy instead of adding a new long CSS class

Reply:
@maryamhdr 
any thoughts?

Reply:
yes, I think we can use the nested stylings...